indico/MaKaC/webinterface/common/abstractDataWrapper.py

```diff
@@ -308,6 +308,8 @@ def _checkParams(self, params, conf, headerSize):
             display_values = True
 
         self._abstractData = AbstractData(conf.getAbstractMgr(), params, headerSize, displayValues=display_values)
+        self._doNotSanitizeFields = self._abstractData.getFieldNames()
```

Comment:
It returns a freshly created list, but that's a bit dangerous since you modify it afterwards. It'd rather do e.g. this:

    self._doNotSanitizeFields = self._abstractData.getFieldNames() + ['title']

That way you are sure you'll never have side effects.